nomos-services/cryptarchia-consensus/src/lib.rs

```diff
@@ -446,6 +469,17 @@ where
                         .await;
                     }
 
+                    Some(mixed_msg) = incoming_mixed_msgs.next() => {
+                        match wire::deserialize::<Block<ClPool::Item, DaPool::Item>>(&mixed_msg) {
+                            Ok(block) => {
+                                adapter.broadcast(NetworkMessage::Block(block)).await;
```

Comment:
Nit: Now that we have two different networks, this `adapter` could use a more descriptive name.

Reply:
Good point! Fixed in 9ab301a92cb2b4f38e4b06f935a98d0b8c426075